src/test/rgw/rgw_multi/tests.py

```diff
@@ -387,6 +395,26 @@ def set_master_zone(zone):
     log.info('Set master zone=%s, waiting %ds for reconfiguration..', zone.name, config.reconfigure_delay)
     time.sleep(config.reconfigure_delay)
 
+def enable_bucket_sync(zone, bucket_name):
+    cmd = ['bucket', 'sync', 'enable', '--bucket', bucket_name] + zone.zone_args()
+    zone.cluster.admin(cmd)
+
+def disable_bucket_sync(zone, bucket_name):
+    cmd = ['bucket', 'sync', 'disable', '--bucket', bucket_name] + zone.zone_args()
+    zone.cluster.admin(cmd)
+
+def check_buckets_sync_status_obj_not_exist(zone, buckets):
+    cmd = ['log', 'list'] + zone.zone_arg()
+    for _ in range(config.checkpoint_retries):
+        log_list, ret = zone.cluster.admin(cmd, check_retcode=False)
```

Comment:
* could you move the `cmd =` assignment inside the for loop, so teuthology doesn't keep appending extra `--cluster c1 --debug-rgw 0` args?
* add a `read_only=True` arg for `zone.cluster.admin()`